src/util/codec/mysql/json/json_type.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::Json;
+
+const JSON_TYPE_BOOLEAN: &[u8] = b"BOOLEAN";
+const JSON_TYPE_NONE: &[u8] = b"NULL";
```

Comment:
Since we  only need `binary` in `tikv`. And if we use  `str` here, it would still be converted to `binary` after `json_type`  @siddontang